_templates/layout.html

```diff
@@ -0,0 +1,10 @@
+{# Import the theme's layout. #}
```

Comment:
There is a ``layout.html`` which is default for sphinx (). ``_theme/pydy/layout.html`` overrides it which is here defining our new layout whereas ``_templates/layout.html`` overrides/adds to layout.html of the theme. This is how sphinx works.